src/data/data.json

```diff
@@ -0,0 +1,255 @@
+{
+  "tasks": [
+    {
+      "id": "1",
+      "name": "Prepare Lesson Plan",
+      "description": "Create a comprehensive lesson plan for the upcoming semester.",
+      "dueDate": "2024-07-15T23:59:59.000Z",
+      "priority": 5,
+      "subtasks": [
+        {
+          "id": 1,
+          "name": "Research Curriculum",
+          "description": "Review and analyze the curriculum to identify key topics.",
+          "Completed": true
+        },
+        {
+          "id": 2,
+          "name": "Draft Weekly Plans",
+          "description": "Create a draft plan for each week's lessons.",
+          "Completed": true
+        },
+        {
+          "id": 3,
+          "name": "Create Assessment Materials",
+          "description": "Design quizzes and tests for student evaluations.",
+          "Completed": false
+        }
+      ],
+      "notes": [
+        "Ensure all lesson plans align with the updated curriculum guidelines.",
```

Comment:
@Haydox01 the ui includes display of date note was added. I recommend the notes field be an array of objects where each object has the keys "note" and "createdAt".